Doc/library/importlib.rst

```diff
@@ -1241,6 +1241,74 @@ find and load modules.
     and how the module's :attr:`__file__` is populated.
 
 
+.. class:: AppleFrameworkLoader(fullname, dylib_path, parent_paths=None)
+
+   A specialization of :class:`importlib.machinery.ExtensionFileLoader` that
+   is able to load extension modules in Framework format.
+
+   For compatibility with the iOS App Store, *all* binary modules in an iOS app
+   must be ``.dylib objects``, contained in a framework with appropriate
+   metadata, stored in the ``Frameworks`` folder of the packaged app. There can
+   be only a single binary per framework, and there can be no executable binary
+   material outside the Frameworks folder.
+
+   If you're trying to run ``from foo.bar import _whiz``, and ``_whiz`` is
+   implemented with the binary module ``foo/bar/_whiz.abi3.dylib`` (or any
+   other ABI .dylib extension), this loader will look for
+   ``{sys.executable}/Frameworks/foo.bar._whiz.framework/_whiz.abi3.dylib``
+   (forming the package name by taking the full import path of the library,
+   and replacing ``/`` with ``.``).
+
+   However, this loader will re-write the ``__file__`` attribute of the
+   ``_whiz`` module will report as the original location inside the ``foo/bar``
```

Comment:
```suggestion
   ``_whiz`` module to report the original location inside the ``foo/bar``
```

Reply:
I've reworked this sentence, so the suggestion no longer applies.